pylib/cqlshlib/cqlshmain.py

```diff
@@ -277,22 +276,10 @@ def __repr__(self):
         return '<%s %s>' % (self.__class__.__name__, self.message())
 
 
-def maybe_ensure_text(val):
```

Comment:
Since Python 3.0, all string are Unicode.  It appears left over from python 2.x.  Its used only on four command line args to "make sure user values .. are in unicode"